Signal/src/ViewControllers/OWSConversationSettingsTableViewController.m

```diff
@@ -222,11 +222,12 @@ - (void)updateTableContents
                             if (!strongSelf) {
                                 return;
                             }
-                            FingerprintViewController *fingerprintViewController = [[UIStoryboard main]
-                                instantiateViewControllerWithIdentifier:@"FingerprintViewController"];
+                            FingerprintViewController *fingerprintViewController = [FingerprintViewController new];
                             [fingerprintViewController configureWithRecipientId:strongSelf.thread.contactIdentifier];
                             fingerprintViewController.dismissDelegate = strongSelf;
-                            [strongSelf presentViewController:fingerprintViewController animated:YES completion:nil];
+                            UINavigationController *navigationController =
+                                [[UINavigationController alloc] initWithRootViewController:fingerprintViewController];
+                            [strongSelf presentViewController:navigationController animated:YES completion:nil];
```

Comment:
Previously the fingerprint view was quite inconsistent with the rest of our UI.  It was "white text-on-dark background" - I've inverted the colors for consistency.  It also was semi-transparent. I've removed that.